### PR TITLE
feat: allow configuring the name/path of ffprobe

### LINKED ISSUE #21: Add a `Config` option for the `ffprobe` path
This would give users a way to not rely on `$PATH` to find `ffprobe`.

### PR DESCRIPTION
**Pull request by lovesegfault**
Fixes: #21

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -71,6 +72,7 @@ pub fn ffprobe_config(
 #[derive(Clone, Debug)]
 pub struct Config {
     count_frames: bool,
+    ffprobe_bin: String,
```
Comment (theduke) [inline src/lib.rs]: Let's make this a `std::path::PathBuf` instead.

**Review by theduke — CHANGES_REQUESTED** [review]
Thanks!

Just one small change.

File: src/lib.rs (hunk 1 above)
Comment (lovesegfault) [reply src/lib.rs]: I thought about that, but the thing is a bin name (e.g. `ffprobe-custom`) which should be found via `$PATH` isn't really a `PathBuf`, semantically. You can flip this argument, and say that an explicit path (e.g. `/opt/bin/my-ffprobe`) isn't really a `String`, I guess.

I think the best answer might be storing a `PathBuf` but make the builder method take `impl AsRef<Path>` so you can still pass `"bin-name"`.

Comment (lovesegfault) [reply src/lib.rs]: Done :)

Comment (theduke) [reply src/lib.rs]: Yeah, I guess if we are pedantic an `OsString` would be the most generic way to go, but I think PathBuf is fine.

**Review by theduke — APPROVED** [review]

**Pull request merged**